src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkBase.cs

```diff
@@ -8,64 +8,31 @@ namespace Xamarin.Android.Build.Utilities
 {
 	abstract class MonoDroidSdkBase
 	{
-		protected readonly static string DebugRuntime = "Mono.Android.DebugRuntime-debug.apk";
 		protected readonly static string ClassParseExe = "class-parse.exe";
-		protected readonly static string GeneratorScript = "generator";
-
-		// I can never remember the difference between SdkPath and anything else...
-		[Obsolete ("Do not use.")]
-		public string SdkPath { get; private set; }
```

Comment:
A note of all the removals here. Doing this we break API compatibility with android-tools. Should these changes been pushed to the xamarin-android-tools repo? 
AFAIK the end goal is to move all the SDK resolving code to xamarin-android-tools and have `Xamarin.Android.Build.Utilities` and android-tools use that so we have one common code base.